daffodil-test/src/test/resources/org/apache/daffodil/section13/text_number_props/TextNumberProps.tdml

```diff
@@ -1520,49 +1520,31 @@
     </tdml:infoset>
 
   </tdml:parserTestCase>
-  
-  <!--
-     Test Name: textStandardDecimalSeparator10
-        Schema: textNumberPattern
-          Root: tnp26
-       Purpose: This test demonstrates that textStandardDecimalSeparator can be a whitespace-separated list of
-                single characters that appear (individually) in the data as the decimal separator
-  -->
```

Comment:
Even though this comment and the next one were outdated, why were they deleted instead of updated?

Reply:
I believe these block comments, which just restate what is obvious from the test itself, serve no purpose. 

I will change the test names however, to better reflect the specific thing they are testing.